src/mixins/legal-api-mixin.ts

```diff
@@ -36,7 +36,7 @@ export default class LegalApiMixin extends Vue {
    * @param filing the object body of the request
    * @returns a promise to return the filing
    */
-  createCorrection (businessId: string, filing: CorrectionFilingIF): Promise<any> {
+  async createCorrection (businessId: string, filing: CorrectionFilingIF): Promise<any> {
```

Comment:
I checked, and the places where these 2 methods are called already "awaited" them.
I'm not sure why no code error was displayed to say that were were awaiting something that wasn't async.